gulpfile.js

```diff
@@ -26,40 +26,63 @@ var browserifyOptions = {
     bundleExternal: false
 };
 
-gulp.task('bower', function(cb){
-    bower.commands.install().on('end', function (installed){
+gulp.task('bower', function(cb) {
+    bower.commands.install().on('end', function(installed) {
         console.log(installed);
         cb();
     });
 });
 
 gulp.task('clean', ['lint'], function(cb) {
-    del([ DEST ], cb);
+    del([DEST], cb);
 });
 
-gulp.task('lint', function(){
+gulp.task('lint', function() {
     return gulp.src(['./*.js', './lib/*.js'])
         .pipe(jshint())
         .pipe(jshint.reporter('default'));
 });
 
-gulp.task('build', ['clean'], function () {
+gulp.task('clean-doc', function(cb) {
+    del(['doc'], cb);
+});
+
+gulp.task('jsdoc', ['clean-doc'], function() {
+    var template = {
+        path: 'ink-docstrap',
+        systemName: pjson.name,
+        footer: '',
+        copyright: 'Ethereum',
+        navType: 'vertical',
+        theme: 'cerulean',
+        linenums: true,
+        collapseSymbols: false,
+        inverseNav: false
+    };
+
+    gulp.src(['./lib/*.js', 'README.md', './package.json'])
```

Comment:
First, you are right with the **

Second, I started the changes required to get more in the doc but I first wanted to check with you. One main thing to do is to go away from the /// comments and use /** comments.

In a previous PR, I mentioned actually that I just plugged things together for now and I was asking you if you had any plan on using something else then jsdoc, which would explain why you use ///

So if you are fine with /** instead of ///, we could go ahead.